portfolio/src/main/webapp/script.js

```diff
@@ -15,58 +15,68 @@
 getComments(5);
 
 // Submits a comment to be stored in the server.
-document.querySelector('.comment-form').addEventListener('submit', async function(e) {
-  e.preventDefault();
-  const response = await fetch('/comment', {
-    method: 'POST',
-    body: new URLSearchParams(new FormData(e.target)),
+document
+  .querySelector(".comment-form")
```

Comment:
use single quotes for strings: https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes